feature-libs/customer-ticketing/core/facade/customer-ticketing.service.ts

```diff
@@ -321,8 +328,12 @@ export class CustomerTicketingService implements CustomerTicketingFacade {
     return this.createTicketEventCommand.execute(ticketEvent);
   }
 
-  uploadAttachment(file: File | null, eventCode: string): Observable<unknown> {
-    return this.uploadAttachmentCommand.execute({ file, eventCode });
+  uploadAttachment(
+    file: File | null,
```

Comment:
file shouldn't be null

Reply:
Fixed https://github.com/SAP/spartacus/pull/16383/commits/67f773742f3c238689ded8a66069c9fafa878ebc